<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomerForm</class>
 <widget class="QDialog" name="AddCustomerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="afmLabel">
      <property name="text">
       <string>AFM</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="afmLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="phoneLabel">
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="phoneLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="addressLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="surnameLabel">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="surnameLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="idnumberLabel">
      <property name="text">
       <string>Id-Number</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="idnumberLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="passowordLabel">
      <property name="text">
       <string>Passoword</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="passowordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
